packages/bpk-component-dialog/src/BpkDialog.scss

```diff
@@ -29,6 +29,12 @@
     }
   }
 
+  &--with-flare {
+    height: $bpk-spacing-base * 10;
```

Comment:
This can be overridden by consumers, right?

Reply:
Yup using `flareClassName` it will allow consumers to override all of the styles that are set here

Reply:
For reference for reviewers https://github.com/Skyscanner/backpack/blob/28c9ee095a27cf2fb58920dcefaa9d34ead5679f/packages/bpk-component-dialog/src/BpkDialog.js#L67